### ISSUE TITLE
[Adaptive cards> Designer> Card Payload Editor]: Name is not defined for 'Card payload Editor' button.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7917]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25197.1000)
Browser: Edge Dev (107.0.1379.1)
URL: https://adaptivecards.io/designer/
Screen reader: Narrator

**Repro steps:**

1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through 'Ctrl+ Windows+ Enter key.
3. Navigate to the 'Designer' tab through tab key and activate it.
4. Navigate to the 'Card Payload Editor' button through tab key.
5. Observe the issue.

**Actual Result:**
On Designer tab, Name is not defined for 'Card Payload Editor'. Screen reader is announcing as **'Button Collapsed Show card payload editor'.**

**Expected Result:**
On Designer tab, proper name should be defined for 'Card Payload Editor'. Screen reader should announce as **Card payload editor Button Collapsed Show card payload editor'.**

**Notes:**
Same issue repro with 
1. 'Sample Data Editor', 'Card Structures', 'Element Properties' and 'Feedback' button.

**User Impact:**
Screen reader user will get impacted as user will not be able to understand the purpose of the content and will not be able to navigate or track the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104483005/191720807-7711cb27-255b-4bf0-8b21-d90b6ac15273.mp4

<img width="960" alt="Code Snipped~ Card payload editor name is not defined" src="https://user-images.githubusercontent.com/104483005/191721180-08b1db05-a60e-4af2-bcea-f61647d516f9.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is working fine on the environment below, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
